release_checklist.md

```diff
@@ -0,0 +1,8 @@
+- [ ] Create SQL Release patch into SQL release directory and move cleanup SQL patches
+- [ ] composer update ?
```

Comment:
shouldn't this be done before release testing?

Reply:
sure we can reorganize this list